### DISCUSSION TITLE
How to integrate onesignal with directus? Need help

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#15382] [Archive]
i have added OneSignalSDKWorker.js here

![image](https://user-images.githubusercontent.com/36895131/188265199-6e24a972-5017-41bc-b070-e8a40cc44e17.png)

html code in project settings,

Is there anything else to be done

![image](https://user-images.githubusercontent.com/36895131/188265266-99a5e889-e568-4c57-9d88-37ebcef1c234.png)

**Comment by MiniDigger** [comment]
take a look at custom hooks: https://docs.directus.io/extensions/hooks.html

**Reply by Jeyakumarrathnaselvi** [reply]
Can someone help me here. Not able to implement

**Reply by br41nslug** [reply]
It is not possible to include arbitrary javascript globally in directus at this time. https://github.com/directus/directus/discussions/12266

**Reply by Jeyakumarrathnaselvi** [reply]
How to implement a push notification with directus then . pls guide

**Reply by br41nslug** [reply]
Pushing notifications shouldnt be a problem using hooks as MiniDigger mentioned. For receiving push notifications that is currently not possible within the App.

**Comment by anjana096** [comment]
@Jeyakumarrathnaselvi did you found any way to send push notifications? @br41nslug what is the current status of receiving push notifications?
